plugins/hls-semantic-tokens-plugin/src/Ide/Plugin/SemanticTokens/SemanticConfig.hs

```diff
@@ -0,0 +1,95 @@
+{-# LANGUAGE GADTs             #-}
+{-# LANGUAGE OverloadedLabels  #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+{-# LANGUAGE TypeFamilies      #-}
+{-# LANGUAGE TypeOperators     #-}
+
+module Ide.Plugin.SemanticTokens.SemanticConfig where
```

Comment:
Is this less code than defining them manually? ;) It's only 11 items. Up to you though :shrug: